README.md

```diff
@@ -41,47 +46,79 @@ See the [changelog](CHANGELOG.md) for the latest release. You might also find us
+* `xml.format.enforceQuoteStyle`: Enforce `preferred` quote style (set by `xml.preferences.quoteStyle`) or `ignore` quote style when formatting. Default is `ignore`. 
+See [here](https://github.com/redhat-developer/vscode-xml/blob/master/docs/Formatting.md#xmlformatenforcequotestyle) for more information.
+* `xml.format.joinCDATALines` : Set to `true` to join lines in CDATA content during formatting. Default is `false`. 
+See [here](https://github.com/redhat-developer/vscode-xml/blob/master/docs/Formatting.md#xmlformatjoincdatalines) for more information.
+* `xml.format.joinCommentLines` : Set to `true` to join lines in comments during formatting. Default is `false`. 
+See [here](https://github.com/redhat-developer/vscode-xml/blob/master/docs/Formatting.md#xmlformatjoincommentlines) for more information.
+* `xml.format.joinContentLines` : Normalize the whitespace of content inside an element. Newlines and excess whitespace are removed. Default is `false`. See [here](https://github.com/redhat-developer/vscode-xml/blob/master/docs/Formatting.md#xmlformatjoincontentlines) for more information.
+* `xml.format.preserveAttributeLineBreaks`: Preserve line breaks that appear before and after attributes. This setting is overridden if `xml.format.splitAttributes` is set to `true`. Default is `false`. 
+See [here](https://github.com/redhat-developer/vscode-xml/blob/master/docs/Formatting.md#xmlformatpreserveattributelinebreaks) for more information.
+* `xml.format.preserveEmptyContent`: Preserve empty content/whitespace in a tag. Default is `false`. 
+See [here](https://github.com/redhat-developer/vscode-xml/blob/master/docs/Formatting.md#xmlformatpreserveemptycontent) for more information.
+* `xml.format.preservedNewLines`: Preserve new lines that separate tags. The value represents the maximum number of new lines per section. A value of 0 removes all new lines. Default is `2`. 
+See [here](https://github.com/redhat-developer/vscode-xml/blob/master/docs/Formatting.md#xmlformatpreservednewlines) for more information.
+* `xml.format.spaceBeforeEmptyCloseTag`: Insert space before end of self closing tag. \nExample:\n  ```<tag/> -> <tag />```. Default is `true`. 
+See [here](https://github.com/redhat-developer/vscode-xml/blob/master/docs/Formatting.md#xmlformatspacebeforeemptyclosetag) for more information.
+* `xml.format.splitAttributes` : Split multiple attributes each onto a new line. Default is `false`. 
+See [here](https://github.com/redhat-developer/vscode-xml/blob/master/docs/Formatting.md#xmlformatsplitattributes) for more information.
 * `xml.preferences.quoteStyle`: Preferred quote style to use for completion: `single` quotes, `double` quotes. Default is `double`.
+
 * `xml.autoCloseTags.enabled` : Enable/disable autoclosing of XML tags. Default is `true`. \n\nIMPORTANT: Turn off `#editor.autoClosingTags#` for this to work.
```

Comment:
The \n are more places to fix, see https://github.com/redhat-developer/vscode-xml/pull/358

Reply:
Ooops sorry I have not taken time to see your PR, thanks. Fixed.